src/components/SeeMore.tsx

```diff
@@ -1,7 +1,25 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { SeeMoreProps } from './../interfaces'
 
-export default (props: SeeMoreProps) => {
+export default function seeMore(props: SeeMoreProps) {
+    useEffect(() => {
+        document.addEventListener("keydown", handleKeyDown);
+        return () => {
+            document.removeEventListener("keydown", handleKeyDown);
+        }
+    }, [])
+
+    const handleKeyDown = (e: KeyboardEvent) => {
+        if (e.key === 'ArrowUp') {
+            e.preventDefault()
+            props.toggleMore(true)
+        }
+        else if (e.key === 'ArrowDown') {
```

Comment:
Can we also add `Esc` for the escape key here? Would make sense in addition to the ArrowDown.

Reply:
I also added support for Esc key.